etc/hub.bash_completion.sh

```diff
@@ -382,6 +382,15 @@ EOF
   }
 
   # Enable completion for hub even when not using the alias
-  complete -o bashdefault -o default -o nospace -F _git hub 2>/dev/null \
-    || complete -o default -o nospace -F _git hub
+  if declare -F _git >/dev/null; then
+    # git < 2.30
+    complete -o bashdefault -o default -o nospace -F _git hub 2>/dev/null \
+      || complete -o default -o nospace -F _git hub
+  elif declare -F __git_main; then
```

Comment:
```suggestion
  elif declare -F __git_main >/dev/null; then
```

Consistent with the preceding conditional branch, this fixes `__git_main` being echoed during shell session initialization.

Reply:
Oops, yes.

Reply:
In fact, in my local git clone it turns out I had already made this change!! and used it locally. The problem is, that I forgot to `git add` it...